packages/coinstac-simulator/src/file-loader.js

```diff
@@ -0,0 +1,110 @@
+'use-strict';
+
+const glob = require('glob');
+const loadCSV = require('csv-load-sync');
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+
+/**
+ * syncronously loads and parses files from a globbing pattern, and concats the data
+ * into an array
+ * @param  {string} fileGlob glob pattern for input files
+ * @param  {string} delim    delimiter used for txt files, defualts to ','
+ * @return {array}          array of parsed input data
+ */
+const loadFiles = (fileGlob, delim) => {
+  const delimiter = delim || ',';
+  const files = glob.sync(fileGlob);
+  let userData = [];
+  files.forEach((filePath) => {
+    if (fs.lstatSync(filePath).isFile()) {
+      const data = fs.readFileSync(filePath, 'utf8');
+      const ext = path.extname(filePath);
+      if (ext === '.json') {
+        // expects JSON array
+        userData = userData.concat(JSON.parse(data));
+      } else {
+        if (delimiter.match(/\n|\r/g)) {
+          userData = userData.concat(data.replace(/\n$|\r$/, '').split(delimiter));
+        } else {
+          userData = userData.concat(data.replace(/\n|\r/g, '').split(delimiter));
+        }
+      }
+    }
+  });
+  return userData;
```

Comment:
This also does no casting, so if you have an 'csv' array, it'll turn up as an array of strings vs a json array will correctly be integers/floats. I think we should cast, but I'd like feedback